Exercises/4-count-types.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
-const countTypesInArray = null;
+const arr = [
+  1, 'Kiev', true, -100, undefined,
+  false, { name: 'ser' }, 20, 'Warrior life', true
+];
+
+const countTypesInArray = arr => {
+  const obj = {};
+  for (const v of arr) {
+    obj[typeof v] === undefined ?
+      obj[typeof v] = 1 :
+      ++obj[typeof v];
+  }
+  return obj;
+};
+
+console.log(countTypesInArray(arr));
```

Comment:
Remove debug output

Reply:
Put repeating expressions to local identifiers, for example: `typeof v` and `obj[typeof v]`

Reply:
done